src/status_im/subs/wallet/wallet.cljs

```diff
@@ -443,27 +451,31 @@
  :wallet/current-viewing-account-tokens-filtered
  :<- [:wallet/current-viewing-account-or-default]
  :<- [:wallet/network-details]
+ :<- [:wallet/tokens-supported-chains]
  :<- [:wallet/wallet-send]
- (fn [[account networks send-data] [_ {:keys [query chain-ids hide-token-fn]}]]
+ (fn [[account networks token-supported-chains send-data] [_ {:keys [query chain-ids hide-token-fn]}]]
    (let [tx-type       (:tx-type send-data)
-         tokens        (->> (:tokens account)
-                            (map
-                             (fn [token]
-                               (assoc token
-                                      :bridge-disabled?  (and (= tx-type :tx/bridge)
-                                                              (send-utils/bridge-disabled? (:symbol
-                                                                                            token)))
-                                      :networks          (cond->> (network-utils/network-list token
-                                                                                              networks)
-                                                           chain-ids
-                                                           (filter #(some #{(:chain-id %)} chain-ids)))
-                                      :available-balance (utils/calculate-total-token-balance token)
-                                      :total-balance     (utils/calculate-total-token-balance
-                                                          token
-                                                          chain-ids))))
-                            (filter (fn [{:keys [networks]}]
-                                      (pos? (count networks))))
-                            (remove #(when hide-token-fn (hide-token-fn constants/swap-tokens-my %))))
+         tokens        (->>
```

Comment:
the usual Clojure code style for `->>` is putting the threaded var next to `->>`:

```clojure
;; Do:
(->> a
     (f)
     (g))
;; Don't:
(->> 
  a
  (f)
  (g))
```